bayesian/api_v1.py

```diff
@@ -293,6 +298,11 @@ def post():
             version = dependency.get('version')
             if not all([ecosystem, package, version]):
                 raise HTTPError(422, "provide the valid input.")
+            if check_for_accepted_ecosystem(ecosystem) is False:
```

Comment:
`if not check_for_accepted_ecosystem(ecosystem):`

Reply:
changed